### PR TITLE
Implemented checkbox

### PR DESCRIPTION
**Pull request by bdlukaa**

### CODE REVIEW

**Comment by GroovinChip** [comment]
@bdlukaa pull the latest changes from dev

**Comment by bdlukaa** [comment]
I'll do it as soon as I get home

File: lib/src/buttons/checkbox.dart [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        height: size,
+        width: size,
+        alignment: Alignment.center,
+        decoration: isDisabled || value == null || value == true
+            ? BoxDecoration(
+                color: CupertinoDynamicColor.resolve(
+                  isDisabled
+                      ? disabledColor
+                      : activeColor ??
+                          context.maybeStyle?.accentColor ??
```
Comment (GroovinChip) [inline lib/src/buttons/checkbox.dart]: There is no `maybeStyle` any more.  Consider `activeColor ?? context.macosTheme.accentColor!` instead.

File: lib/macos_ui.dart [hunk 2]
```diff
@@ -2,7 +2,8 @@ library macos_ui;
```
Comment (GroovinChip) [inline lib/macos_ui.dart:2]: Run `dartfmt` and optimize imports on this file

File: lib/src/buttons/checkbox.dart [hunk 3]
```diff
@@ -0,0 +1,102 @@
+import 'package:macos_ui/macos_ui.dart';
```
Comment (GroovinChip) [inline lib/src/buttons/checkbox.dart:1]: Run `dartfmt` on this file

**Review by GroovinChip — CHANGES_REQUESTED** [review]

File: lib/src/buttons/checkbox.dart (hunk 1 above)
Comment (bdlukaa) [reply lib/src/buttons/checkbox.dart]: Done

File: lib/macos_ui.dart (hunk 2 above)
Comment (bdlukaa) [reply lib/macos_ui.dart:2]: I'll create a pr organizing this file after my other 3 prs get merged

**Review by GroovinChip — CHANGES_REQUESTED** [review]
The checkbox is not visible in light theme. 
Screen Shot 2021-04-18 at 10 22 28 AM
Use the spec from the Figma kit to ensure the UI is correct in each theme
Screen Shot 2021-04-18 at 10 24 45 AM

File: example/pubspec.lock [hunk 4]
```diff
@@ -73,6 +73,7 @@ packages:
       path: ".."
       relative: true
     source: path
+    version: "0.0.3"
```
Comment (GroovinChip) [inline example/pubspec.lock]: Can you check what version is in your pubspec.yaml please? When I check out your branch I have issues running `flutter pub get`. I think yours still says 0.0.3.

**Review by GroovinChip — CHANGES_REQUESTED** [review]

File: example/pubspec.lock (hunk 4 above)
Comment (bdlukaa) [reply example/pubspec.lock]: Fixed it

Comment (GroovinChip) [reply example/pubspec.lock]: Ok, I'll check it out when I'm back online later

**Review by GroovinChip — CHANGES_REQUESTED** [review]
We really ought to get that gallery app off the ground so we stop running into conflicts in the example 😅

@bdlukaa Maybe it would be a good idea to consolidate this PR and #27 into one for easier conflict resolution?

**Comment by bdlukaa** [comment]
@GroovinChip I think it's better to create a new pr, after this and the others get merged, with the example app updated

@bdlukaa oh, for sure. The gallery app can be done after these get merged. That said, I still think it's worth making a combined PR for checkbox and radio button to reduce conflicts

**Comment by bdlukaa** [comment]
Closing as duplicate of https://github.com/GroovinChip/macos_ui/pull/47

**Pull request closed without merging**